sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InterfaceMethodsShouldBeCallableByChildTypes.CSharp9.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Tests.Diagnostics
```

Comment:
pedantic: I think we can skip namespaces from most of the tests (if we remove all the boilerplate code the test cases will get a bit more visible)